src/app/core/component/markdown-content-container/markdown-pattern-sectioncontent/markdown-pattern-section-content.component.ts

```diff
@@ -71,6 +72,7 @@ export class MarkdownPatternSectionContentComponent extends DataRenderingCompone
   changeText(value: string): void {
     this.markdownDiv.nativeElement.innerHTML = '';
     this.renderedData = value;
+    this.showSection = this.renderedData.length > 0
```

Comment:
Just to ensure that there will be no NPE, maybe add ´this.showSection = this.renderedData && this.renderedData.length > 0;´